pkg/apiserver/apiserver.go

```diff
@@ -365,6 +365,8 @@ func (s *APIServer) waitForResourceSync(stopCh <-chan struct{}) error {
 		{Group: "devops.kubesphere.io", Version: "v1alpha1", Resource: "s2ibuildertemplates"},
 		{Group: "devops.kubesphere.io", Version: "v1alpha1", Resource: "s2iruns"},
 		{Group: "devops.kubesphere.io", Version: "v1alpha1", Resource: "s2ibuilders"},
+		{Group: "devops.kubesphere.io", Version: "v1alpha3", Resource: "devopsprojects"},
```

Comment:
Why crd named `devopsproject`, and `devops` in restful API spec?

Reply:
The CRD 's name is devopsproject, but there are some old /kapis using /devops/{devops} path.
The consistent url path is good for the convenience of devops authentication. 
/cc @wansir

Reply:
Why we name the crd `devops` if for compatibility reason? It's more clear

Reply:
Devopsproject is used as the name of CRD during the refactoring phase. Here just keep consistense.